### PR TITLE
Add a rule for minimum required phony targets

### PR DESCRIPTION
**Pull request by meatballhat**
with some goimports formatting along for the ride :grin:

### CODE REVIEW

**Comment by mrtazz** [comment]
Nah this is fine with me right now. Sorry I was heads down writing the config stuff. Unfortunately you have to merge again :\.

**Comment by meatballhat** [comment]
I'm deconflicting, yep! :smile_cat:

**Comment by mrtazz** [comment]
I'm working on fixing the travis build in master. But I'll merge it locally in a bit and test it and push it out.

File: Makefile [hunk 1]
```diff
@@ -33,15 +33,17 @@ INSTALLED_MAN_TARGETS = $(addprefix $(PREFIX)/share/man/man1/, $(MAN_TARGETS))
...
 DEPDIR = .d
-$(shell install -d $(DEPDIR))
 MAKEDEPEND = echo "$@: $$(go list -f '{{ join .Deps "\n" }}' $< | awk '/github/ { gsub(/^github.com\/[a-z]*\/[a-z]*\//, ""); printf $$0"/*.go " }')" > $(DEPDIR)/$@.d
 
 $(DEPDIR)/%.d: ;
 .PRECIOUS: $(DEPDIR)/%.d
 
+$(DEPDIR):
+	install -d $@
+
```
Comment (mrtazz) [inline Makefile]: did you run into problems with this not being a rule? If I recall correctly this was intentional to solve some chicken and egg problem.

Comment (meatballhat) [inline Makefile]: Ah sorry, I should have talked it through.  I hit issues with the bare `$(shell install -d $(DEPDIR))` in combination with explicit `make all`.

Comment (mrtazz) [inline Makefile]: ah ok. Yea I'm fine with moving it to a rule. Was just curious what the reason was. We'll see if we run into the other problem again in the future.

**Comment by meatballhat** [comment]
I'll stand back and watch the magic unfold ✨

**Comment by meatballhat** [comment]
Or, rather, I'll gladly resolve conflicts and re-push whenever ye like :smile_cat:

**Pull request closed without merging**